recipe/migrations/cuda130.yaml

```diff
@@ -0,0 +1,64 @@
+migrator_ts: 1755016036
+__migrator:
+  use_local: true
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    0
+  paused: false
+  override_cbc_keys:
+    - cuda_compiler_stub
+  check_solvable: false
+  primary_key: cuda_compiler_version
```

Comment:
Speaking of the zip, we will need to modify the global cuda zip to include `c_stdlib_version`, and then override that value in the migrator.